### PR TITLE
Call the right Jolie executor on OS Windows

### PR DESCRIPTION
**Pull request by mwallnoefer**

### CODE REVIEW

File: .idea/compiler.xml [hunk 1]
```diff
@@ -66,7 +65,7 @@
       <module name="joliec" target="1.8" />
       <module name="jsonrpc" target="1.8" />
       <module name="launchers" target="1.5" />
-      <module name="libjolie" target="11" />
+      <module name="libjolie" target="1.8" />
```
Comment (fmontesi) [inline .idea/compiler.xml:68]: Not sure about these changes. I remember at least that we're using APIs in `Optional` that require Java 11. It's very strange that this works... perhaps target is ignored?

File: .idea/compiler.xml [hunk 2]
```diff
@@ -50,9 +50,8 @@
       <module name="http" target="1.8" />
       <module name="https" target="1.8" />
       <module name="javascript" target="1.8" />
-      <module name="jolie" target="11" />
+      <module name="jolie" target="1.8" />
       <module name="jolie-cli" target="1.8" />
-      <module name="jolie-distribution" target="1.5" />
```
Comment (fmontesi) [inline .idea/compiler.xml:55]: Not sure where this removal comes from?

File: .idea/compiler.xml [hunk 3]
```diff
@@ -50,9 +50,8 @@
       <module name="http" target="1.8" />
       <module name="https" target="1.8" />
       <module name="javascript" target="1.8" />
-      <module name="jolie" target="11" />
+      <module name="jolie" target="1.8" />
```
Comment (fmontesi) [inline .idea/compiler.xml:53]: See other comment about Java 11/1.8.

**Comment by mwallnoefer** [comment]
Okay, so let us do a partial revert of https://github.com/jolie/jolie/commit/bc2ab24978380bea5b908e248fbd4e98b17bef99. The problem was that you mixed two independent changes in one commit 😄 !

**Comment by fmontesi** [comment]
> Okay, so let us do a partial revert of [bc2ab24](https://github.com/jolie/jolie/commit/bc2ab24978380bea5b908e248fbd4e98b17bef99). The problem was that you mixed two independent changes in one commit 😄 !

You're right! Thanks for the extra work. 😅

**Pull request merged**